inventory/host_vars/prod2.openfoodnetwork.org.au/config.yml

```diff
@@ -1,8 +1,23 @@
 ---
 
-domain: prod2.openfoodnetwork.org.au
+domain: openfoodnetwork.org.au
```

Comment:
I would also rename the folder for the sake of consistency.

Reply:
agree, the folder name needs to be updated.
 there's another folder www.openfoodnetwork.org.au...

Reply:
Is there any other benefit than consistency? I did this intentionally because I thought the host is the actual computer we are installing to which is different to the application it is hosting and under which domain names the application is reachable.

When I added prod2, the old host was still alive and I didn't want to touch it's configuration. And when it comes to installing a new host, I would name it prod3. Initially, it would have the domain name prod3.openfoodnetwork.org.au because we can't get a certificate for a domain that points to another server. When everything works, I would change the domain, copy the SSL certificates, database and files. In case anything goes wrong, I may still want the old host to be available.

Do you think that is too complicated and I should just use different branches for dealing with old and new hosts?

Reply:
I am not sure what's my opinion here.  I trust you! :-)

From an external perspective, it would be better to keep it simple and have one single domain to connect to au live server, either www.openfoodnetwork.org.au or openfoodnetwork.org.au would be better.

Reply:
Is it okay if I review this again when moving the server next time? Are you happy to confirm this PR @luisramos0?

Reply:
yes yes :+1: